frontend/test/metabase-smoketest/admin_setup.cy.spec.js

```diff
@@ -0,0 +1,643 @@
+      ).click();
+      cy.findByText("Add").click();
+
+      cy.contains(USERS.admin.username);
+
+      // Check member count
+
+      cy.findAllByText("People") // *** Going to People and then to Groups should be unnecessary
+        .last()
+        .click();
+
+      cy.contains("3 other groups");
+
+      cy.findAllByText("Groups")
+        .first()
+        .click();
+
+      cy.contains("Marketing");
+      cy.get("td")
+        .eq("-2")
+        .contains("2");
+    });
+
+    it("should create new users in different groups", () => {
+      cy.server();
+      cy.route({
+        method: "POST",
+        url: "/api/user",
+      }).as("createUser");
+
+      cy.findAllByText("People")
+        .last()
+        .click();
+
+      // Add new user into 2 groups
+
+      cy.findByText("Add someone").click();
+      cy.findByLabelText("First name").type(new_user.first_name);
+      cy.findByLabelText("Last name").type(new_user.last_name);
+      cy.findByLabelText("Email").type(new_user.username);
+      cy.get(".ModalBody")
+        .find(".Icon-chevrondown")
+        .first()
+        .click();
+      cy.findByText("English").click();
+      cy.contains("Turkish").should("not.exist");
+      cy.get(".ModalBody") // *** I should be able to select using $ cy.findByText("Default"), but cypress says there are multiple instances
+        .find(".Icon-chevrondown")
+        .last()
+        .click();
+      cy.findAllByText("collection") // *** I should be able to select collection without "All" because there's only one instance of "collection"
+        .last()
+        .click();
+      cy.findByText("Marketing").click();
+      cy.findByText("Create").click();
+      cy.wait("@createUser").then(xhr => {
+        cy.wrap(xhr.request.body.password).as("password");
+      });
+      cy.findByText("Done").click();
+
+      // Check new user is in those groups
+
+      cy.contains(new_user.first_name + " " + new_user.last_name);
+      cy.get("td")
+        .eq("-3")
+        .contains("2 other groups");
+
+      cy.findAllByText("Groups") // *** These 6 lines of code should be unnecessary.
+        .first()
+        .click();
+      cy.findAllByText("People")
+        .last()
+        .click();
+
+      cy.findAllByText("Groups")
+        .first()
+        .click();
+      cy.get("td")
+        .eq("-2")
+        .contains("3");
+
+      cy.findByText("Marketing").click();
+
+      cy.contains(new_user.username);
+      cy.contains(USERS.nodata.username).should("not.exist");
+    });
+
+    it("should set up custom maps", () => {
+      cy.findByText("Settings").click();
+      cy.findByText("Maps").click();
+
+      cy.contains("Custom Maps");
+      cy.contains("Groups").should("not.exist");
+
+      // cy.findByText("Add a map").click();
+      // cy.get("input")
+      //   .first()
+      //   .type("Test Map");
+      // cy.get("input")
+      //   .last()
+      //   .type("") // *** type GeoJSON url here
```

Comment:
Let's try to add a GeoJSON url - lmk if you want more info on that.

Reply:
I've done geocoding with mapbox before, but I' haven't used a GeoJSON url before. Is there a particular url you want me to use/make?

Reply:
Use our own, so we don't depend on third-party (besides Github):
`https://raw.githubusercontent.com/metabase/metabase/master/resources/frontend_client/app/assets/geojson/world.json`